### PR TITLE
fix(alerts): Fix page not found alert & UI adjustments

### LINKED ISSUE #16857: Flash messages need a margin
Flash messages need a margin on top so they're not smashed up under the navbar.

sign_in_to_freecodecamp_using_your_email_address___freecodecamp

### LINKED ISSUE #16860: In some situations 404 flash message is too wide
learn_to_code___freecodecamp

The URL I visited to cause this was `http://localhost:3000/en/challenges/javascript-algorithms-and-data-structures-projects/palindrome-checkers` but any invalid challenge URL should produce this effect.

### PR DESCRIPTION
**Pull request by vkWeb**
### Pre-Submission Checklist
- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

### Type of Change
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

### Checklist:
- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #16857
Closes #16860

### Description
- Added adequate margins to flash alerts.
- Page not found alert link fix & UI adjustments.

### CODE REVIEW

**Comment by raisedadead** [comment]
This needs a rebase.

Also instead of this, you should re-use the same look and feel of middle centered UX for user not-found

Like so:
image

File: common/app/Flash/flash.less [hunk 1]
```diff
@@ -13,3 +13,7 @@
   flex: 1 0 0px;
   color: #37474f;
 }
+
+#@{ns}-board {
+  margin-top: 50px;
+}
```
Comment (raisedadead) [inline common/app/Flash/flash.less]: Add newline when ending a file, or configure your editor for it.

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: common/app/Flash/flash.less (hunk 1 above)
Comment (vkWeb) [reply common/app/Flash/flash.less]: @raisedadead Done.

**Comment by raisedadead** [comment]
The three pages have their own purpose, it makes sense to unify the look and feel however not the text. The first two are client rendered, and last is server rendered and hence agnostic of each scenario.

For the purposes of this PR I recommend making the requested changes for the client only, and let us know when ready for QA.

Thanks.

**Comment by vkWeb** [comment]
@raisedadead PR updated as per your request. It is ready for merge.

**Review by raisedadead — APPROVED** [review]

**Comment by QuincyLarson** [comment]
@vkWeb Awesome! This looks great. Thanks for making these more visually consistent!

**Pull request merged**